docs/README.md

```diff
@@ -4,6 +4,32 @@ This doc describes Indoor Routing, including how you can customize it to work fo
 
 > **Looking for ArcGIS Indoors?** ArcGIS Indoors is Esri's solution for indoor mapping, location tracking, and wayfinding; no code required. [Learn more about ArcGIS Indoors](https://www.esri.com/en-us/arcgis/products/arcgis-indoors).
```

Comment:
Maybe this would better serve the document if it was placed above the top h1? Just a suggestion...